README.md

```diff
@@ -134,14 +141,14 @@ Remote is meant to allow people from anywhere (or at least anywhere in a given t
 | Roq.ad | All | Full remote | [Link](https://www.roq.ad/careers/) | ? |
 | Salesforce | Some | Full remote depending on the position | [Link](https://careers.salesforce.com/en/jobs/?search=&location=Remote&pagesize=200#results) | ? |
 | Samsara | Some | Full remote | [Link](https://www.samsara.com/company/careers/) | ? |
+| Scribd | All | Full remote | [Link](https://www.scribd.com/careers) | ? |
 | SecurityScoreCard | Some areas | Full remote | [Link](https://securityscorecard.com/company/careers) | ? |
-| Semmle | Some areas | Full remote | [Link](https://semmle.com/careers) | ? |
+| Semmle | Some areas | Full remote | [Link](https://www.github.careers/careers-home) | ? |
```

Comment:
are you sure about this link?

Reply:
TBH, I think that entry should totally get removed (the entire line) unless we feel the company name "Semmle" is/was popular enough to keep it around.

The current link in the README.md (https://semmle.com/careers) now redirects to a GitHub blog post from 2019 (6 years ago) when you click it, announcing that Semmle was acquired by GitHub. So that link no longer goes to a page of open positions. Because Semmle is now part of GitHub (has been since 2019, apparently), I just swapped the `Job Postings` link for GitHub's.

I'm happy to fully remove that line unless you feel it should be kept around for some reason (like the company name is/was super recognizable or something, but I don't think that's the case, IMO).

Thx!

Reply:
I agree, we can remove that!